fastkafka_gen/_code_generator/prompts.py

```diff
@@ -232,20 +232,48 @@ async def to_output_data(data: float) -> Data:
 
 If the ==== APP DESCRIPTION: ==== section is related to FastKafka but focuses on what is it and its general information then you should respond with 1. 
 
-If from the ==== APP DESCRIPTION: ==== it is NOT possible to infer the topic name you should respond with 2. 
+If from the ==== APP DESCRIPTION: ==== it is NOT possible to infer the topic name or there are NOT any produces/consumes functions defined  you should respond with 2. 
 
-If the ==== APP DESCRIPTION: ==== section is related to FastKafka but focuses how to use it and instructions to create a new app. Also, if AT LEAST one topic AND AT LEAST one produces/consumes is defined, then you should respond with 3. 
+If the ==== APP DESCRIPTION: ==== section is related to FastKafka but focuses how to use it and instructions to create a new app. Also, if AT LEAST one topic AND AT LEAST one produces/consumes functions is defined, then you should respond with 3. 
 
-Here are few examples for category 2 and 3:
-Example 1:
+Here are few examples for categories 2 and 3:
+==== App description 1 ====
 Generate a new fastkafka app, which has a producer function and a consumer function 
-Response 1:
+==== Response 1 ====
 2
+==== Response EXPLAINATION 1 ====
+In the App description 1, user hasn't defined message structure,the names of the consumes and produces functions. Because of this, you should respond with 2 
 
-Example 2:
+==== App description 2 ====
+Create a FastKafka application.
+==== Response 2 ====
+2
+==== Response EXPLAINATION 2 ====
+In the App description 2, user only wrote that he wants a fastkafka app, without any additional information about the logic, messages and consumes/produces function. Because of this, you should respond with 2 
+
+==== App description 3 ====
+create fastkafka app where message has user_data attribute.
+==== Response 3 ====
+2
+==== Response EXPLAINATION 3 ====
+In the App description 3, user wrote that he wants a fastkafka app and defined messages structure, without any additional information about the logic, and consumes/produces function. Because of this, you should respond with 2 
```

Comment:
remove this "consumes/produces function" and add something like "the topic to consume/produce"